apps/web/tsconfig.json

```diff
@@ -1,4 +1,5 @@
 {
+  "extends": "../../tsconfig.json",
```

Comment:
As it was not referred the base configuration was not being picked. Though arguably we might want it to not be. Let me know if that is the intention to roll it back.

Reply:
Oh good catch! it actually a good thing that it picks it. Probably will create some warnings now